src/cron/updateCurrentStreak.ts

```diff
@@ -16,13 +16,16 @@ const cron: Cron = {
       await con.transaction(async (entityManager): Promise<void> => {
         const usersPastStreakTime = await entityManager
           .createQueryBuilder()
-          .select(
-            `us.*, (date_trunc('day', us."lastViewAt" at time zone COALESCE(u.timezone, 'utc'))::date) AS "lastViewAtTz", u.timezone`,
+          .select(`us.*`)
+          .addSelect(
+            `date_trunc('day', us."lastViewAt" at time zone COALESCE(u.timezone, 'utc'))::date`,
+            'lastViewAtTz',
           )
+          .addSelect('u.timezone', 'timezone')
```

Comment:
Just broke the 3 columns into their separate `select` statements to make the columns fetched more obvious.

Reply:
the date_trunc and other things could become a function so we can reuse, since the same thing is used in some other parts outside of cron. Not needed for this PR though.

Reply:
postgresql

Reply:
https://dailydotdev.atlassian.net/jira/software/projects/WT/boards/11/backlog?selectedIssue=WT-2249